mjulialobo/decor-catalog
# feat(nav): add navigation

## Why
We need a navigation for the catalog;

## What was done

- Added full catalog nav;
- Added category pages;
- Adjusted some styles;
- Created NavHeader component;
- Unit tests;

## How to test
- Open the navigation, click on the catalog on the header;
- Try to filter values with the input
- Navigate through the nav links and see if the content changes for each category;
- Test responsiveness;

# Result
![Capturar](https://github.com/mjulialobo/decor-catalog/assets/65983895/3940fd7e-2552-4328-a8d9-0c4a0d74f839)

## Comment
I'll deal with products with no image on the next PR
